docs/getting_started/configuration.rst

```diff
@@ -39,3 +39,41 @@ environment variable can be provided on the command line like so:
 .. code-block:: bash
 
    HTTPS_PROXY=https://localhost:3128 ./prawbot.py
+
+
+Configuring a custom requests Session
+-------------------------------------
+
+PRAW uses `requests`_ to handle
+networking. If your use-case requires custom configuration, it is possible
+to configure a `Session
+<http://docs.python-requests.org/en/master/user/advanced/#session-objects>`_
+and then use it with PRAW.
+
+For example, some networks use self-signed SSL certificates when connecting
+to HTTPS sites. By default, this would raise an exception in Requests. To
+use a self-signed SSL certificate without an exception from Requests, first
+export the certificate as a ``.pem`` file. Then configure PRAW like so:
+
+.. code-block:: python
+
+   import praw
+   from requests import Session
+
+
+   session = Session()
+   session.verify = '/path/to/certfile.pem'
+   reddit = praw.Reddit(requestor_kwargs={'session': session},  # pass Session
```

Comment:
Can we put `requestor_kwargs` in sorted order?

Reply:
Fixed, thanks.